src/main/kotlin/com/strumenta/kolasu/model/Traversing.kt

```diff
@@ -0,0 +1,124 @@
+package com.strumenta.kolasu.model
+
+import java.util.*
+
+/**
+ * Some Kotlinization of Deques used as a stack.
+ */
+typealias Stack<T> = Deque<T>
+
+fun <T> mutableStackOf(): Stack<T> = ArrayDeque()
+fun <T> Stack<T>.pushAll(elements: Collection<T>) {
```

Comment:
nitpick: I would put a blank link between mutableStackOf and pushAll

Reply:
Ok